src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -48,15 +50,27 @@ type ICopyNumberTableWrapperProps = {
     status:"loading"|"available"|"unavailable";
 };
 
-
 @observer
 export default class CopyNumberTableWrapper extends React.Component<ICopyNumberTableWrapperProps, {}> {
-
     public static defaultProps = {
         enableOncoKb: true,
         enableCivic: false
     };
 
+    @computed get hugoGeneSymbolToCytoband() {
+        // build reference gene map
+        const result:{[hugosymbol:string]:string} =
```

Comment:
you should do a check for referenceGenes to be defined here, otherwise could end up with a bug. Unless referenceGenes is mandatory, in which case remove the `?` from the prop.